### PR TITLE
Issue 91: Update Page Notice stories to use EbayPageNoticeTitle

### LINKED ISSUE #91: EbayPageNotice: fix storybook example to add a heading instead of <p>

### PR DESCRIPTION
**Pull request by aaron-nance**
Many of the page-notice examples were using `strong` tags for a header instead of using `EbayPageNoticeTitle`. Updated these stories and their snapshots. 

fixes #91

### CODE REVIEW

File: src/ebay-checkbox/__tests__/index.spec.tsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ describe('<EbayCheckbox>', () => {
                 <EbayCheckbox aria-label="checkbox" onChange={spy} />
             )
             const input = getByLabelText('checkbox')
-            userEvent.click(input)
+            userEvent.click(input);
```
Comment (darkwebdev) [inline src/ebay-checkbox/__tests__/index.spec.tsx:16]: I think we can skip that change :)

File: src/ebay-page-notice/README.md [hunk 2]
```diff
@@ -43,6 +44,8 @@ import "@ebay/skin/page-notice";
 
 | Name         | Type                                                                          | Stateful | Description                                                                                                                                 | Default       |
 | ------------ | ----------------------------------------------------------------------------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------- | ------------- |
-| `status`     | String ( `"general"`,`"attention"`, `"confirmation"`, or `"information"`) | No       | Determines the style and type of notice to be displayed                                                                                     | `"general"` |
+| `status`     | String ( `"general"`,`"attention"`, `"confirmation"`, or `"information"`) | No       | Determines the style and type of notice to be displayed                                                                                         | `"general"`   |
```
Comment (darkwebdev) [inline src/ebay-page-notice/README.md:47]: this change is unnecessary as well

File: src/ebay-page-notice/README.md [hunk 3]
```diff
@@ -43,6 +44,8 @@ import "@ebay/skin/page-notice";
...
 | `aria-label` | String                                                                        | No       | The description of the notice itself for screen readers. Check out [this issue](https://github.com/eBay/skin/issues/1001) for more context. | -             |
 | `children`   | React Node                                                                    | No       | The content to be displayed within the notice                                                                                               | -             |
+| `a11yDismissText` | String                                                                   | No       | Determines if the notice will have a dismiss button.  Acts as the aria-label for the dismiss button. Should not be used with a footer.      | -             |
+| `onDismissed` | Dispatch<boolean>                                                            | No       | Handler for any extra functionality for after a notice is dismissed.                                                                        | -             |
```
Comment (darkwebdev) [inline src/ebay-page-notice/README.md:51]: I think we've renamed it to `onDismiss`?

File: yarn.lock [hunk 4]
```diff
@@ -2592,7 +2592,7 @@
 
 "@testing-library/user-event@^12":
   version "12.8.3"
-  resolved "https://registry.npmjs.org/@testing-library/user-event/-/user-event-12.8.3.tgz#1aa3ed4b9f79340a1e1836bc7f57c501e838704a"
+  resolved "https://registry.yarnpkg.com/@testing-library/user-event/-/user-event-12.8.3.tgz#1aa3ed4b9f79340a1e1836bc7f57c501e838704a"
```
Comment (darkwebdev) [inline yarn.lock:2595]: I don't think it's necessary...

**Review by darkwebdev — CHANGES_REQUESTED** [review]

File: src/ebay-page-notice/README.md (hunk 3 above)
Comment (aaron-nance) [reply src/ebay-page-notice/README.md]: You're right. Forgot to update this when I made the other change. Updated.

File: yarn.lock (hunk 4 above)
Comment (aaron-nance) [reply yarn.lock:2595]: This wasn't something that was done manually. I'm guessing maybe yarn did this when I reverted from v14 back to v12.

File: src/ebay-page-notice/README.md (hunk 2 above)
Comment (aaron-nance) [reply src/ebay-page-notice/README.md]: This is just a whitespace change. Makes things line up more nicely in the editor.

**Comment by aaron-nance** [comment]
Too much extra stuff got pulled in when this was changed to the 2.0.0 branch. Created a new PR for 2.0.0, so closing this.

**Pull request closed without merging**